### PR TITLE
Created main Control

### PR DESCRIPTION
**Pull request by navanchauhan**
I'm working on a menu for update and remove too

**Pull Request Label:**
* [ ] Bug
* [ ] New feature
* [ ] Enhancement
* [x] New component
* [ ] Typo

**Pull Request Checklist:**
- [x] Have you followed the [guidelines for contributing](https://github.com/alexanderepstein/Bash-Snippets/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/alexanderepstein/Bash-Snippets/pulls) for the same fix or component?
- [ ] Have you ran the tests locally with `bats tests`?

-----

### CODE REVIEW

File: bash-snippets.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+}
+
+installationcheck(){
+cd /usr/local/bin
+  for f in cheat cloudup crypt cryptocurrency currency geo lyrics meme movies newton qrify short siteciphers stocks taste todo transfer weather ytview
+    do
+      if [ -e $f ]
+        then
+          echo "$f is Installed"
```
Comment (retokromer) [inline bash-snippets.sh]: `Is Installed` or `is installed`

File: bash-snippets.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        else
+          echo "$f is Not Installed"
```
Comment (retokromer) [inline bash-snippets.sh]: `Is Not Installed` or `is not installed`

File: bash-snippets.sh (hunk 1 above)
Comment (navanchauhan) [reply bash-snippets.sh]: Choice of @alexanderepstein

**Comment by navanchauhan** [comment]
@retokromer converted the text into lowercase

**Comment by retokromer** [comment]
@navanchauhan I guess this is the best solution, as it’s not a title (and a mix is ugly).

**Comment by navanchauhan** [comment]
Also, Codacy thinks that I've wrongly used $ whereas it's important cause it's a text string

File: bash-snippets.sh [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+{
+  httpGet github.com > /dev/null 2>&1 || { echo "Error: no active internet connection" >&2; return 1; } # query github with a get request
+}
+
+header(){
+  COLUMNS=$(tput cols)
+  title="Bash Snippets"
+  installver="Installed Version : $currentVersion"
+  latestver="Latest Version    : $latestVersion"
+  printf "%*s\n" $(((${#title}+$COLUMNS)/2)) "$title"
```
Comment (retokromer) [inline bash-snippets.sh]: `$((${#title}+$COLUMNS)/2)` should be sufficient.

**Comment by alexanderepstein** [comment]
This looks good to me for now, might improve on it in the future but this is good for now, sorry this took me so long I've been extremely busy with work and studying for gres

**Pull request merged**